### ISSUE TITLE
Crash [TwilioConversationsClient initializeClientWithConfig:token:properties:completion:]

### ISSUE THREAD

**Issue by khanhson47** [twilio/conversations-ios#23]
Hi, I'm getting crashes from Crashlytics, I don't know why it's happening and I cannot reproduce it as well.
Please help me check this. Thank you in advance.
![Screen Shot 2022-05-13 at 15 07 22](https://user-images.githubusercontent.com/63218181/168239695-8520d507-9c01-4baf-b261-f6832e3ff855.png)

**Comment by khanhson47** [comment]
It always redirects me to https://support.twilio.com/hc/en-us. How can I create a support ticket?

**Comment by khanhson47** [comment]
Here is how I create the client
```
func loginTwilio(completion: @escaping (Bool) -> Void) {
    getTwilioToken { [weak self] isSuccess in
        guard let self = self else { return }
        guard isSuccess, let token = self.token else {
            completion(false)
            return
        }

        let properties = TwilioConversationsClientProperties()
        TwilioConversationsClient.conversationsClient(withToken: token, properties: properties, delegate: self) { [weak self] result, twilioClient in
            guard let self = self else { return }
            guard result.isSuccessful else {
                completion(false)
                return
            }

            self.client = twilioClient
            if let twilioClient = self.client, twilioClient.user != nil, let deviceTokenData = Defaults[\.deviceTokenData] {
                self.clientRegisterPushNotification(client: twilioClient, token: deviceTokenData)
            }
            completion(true)
        }
    }
}
```

**Comment by Gray-Wind** [comment]
Hello again,

Thank you for the code sample. We are currently investigating the issue now. In the meanwhile I would like to ask you to open a support ticket using this link: http://twilio.com/console/support/tickets/create 

Previous one should work too, but it turned out you need to be logged in to the Console.

**Comment by afanaian01** [comment]
@Gray-Wind Any update on this crash? We're starting to see it on iOS 15.6+.

**Comment by Gray-Wind** [comment]
During the internal investigation we found out that this crash could happen during fast initialization-deinitialization process.

Looking to the code example provided above, I would say it could happen if `loginTwilio` function would be called twice in a short period of time.

We have now a little hot fix in works, it will be published soon.

**Comment by Gray-Wind** [comment]
Hello, take a look at just released version v2.2.5.

**Comment by Gray-Wind** [comment]
Looks like I can close the ticket now, if you have any issues feel free to create a new one.
